hedera-mirror-rest/utils.js

```diff
@@ -1494,6 +1571,77 @@ const convertGasPriceToTinyBars = (gasPrice, hbarsPerTinyCent, centsPerHbar) =>
   return Math.round(Math.max(tinyBars, 1));
 };
 
+const boundToOp = {
+  '(': opsMap.gt,
+  '[': opsMap.gte,
+  ')': opsMap.lt,
+  ']': opsMap.lte,
+};
+const buildTimestampQuery = (range, column, neValues = [], eqValues = [], buildAsIn = true) => {
+  const conditions = [];
+  const params = [];
+
+  if (range) {
+    const bounds = [...range.bounds];
+    if (bounds.length !== 2) {
+      return ['', []];
+    }
+    if (range.begin) {
+      conditions.push(`${column}${boundToOp[bounds[0]]} ?`);
+      params.push(range.begin);
+    }
+    if (range.end) {
+      conditions.push(`${column} ${boundToOp[bounds[1]]} ?`);
+      params.push(range.end);
+    }
+  }
+
+  if (neValues.length) {
+    if (buildAsIn) {
+      conditions.push(`${column} not in (?`.concat(', ?'.repeat(neValues.length - 1)).concat(')'));
+    } else {
+      neValues.forEach((_) => conditions.push(`${column} != ?`));
+    }
+    params.push(...neValues);
+  }
+
+  if (eqValues.length) {
+    if (buildAsIn) {
+      conditions.push(`${column} in (?`.concat(', ?'.repeat(eqValues.length - 1)).concat(')'));
+    } else {
+      eqValues.forEach((_) => conditions.push(`${column} = ?`));
+    }
+    params.push(...eqValues);
+  }
+
+  return [conditions.join(' and '), params];
+};
+
+const buildTimestampRangeQuery = (range, column, neValues = [], eqValues = []) => {
+  const conditions = [];
+  const params = [];
+
+  if (range) {
+    conditions.push(`${column} && ?`);
+    params.push(range);
+  }
+  if (neValues.length) {
+    neValues.forEach((value) => {
+      conditions.push(`not ${column} @> ?`);
+      params.push(Range(value, value, '[]'));
```

Comment:
can just use the value
```suggestion
      params.push(value);
```

Reply:
This would have to be a range because it is using the range operator `@>`